lib/Elastica/Aggregation/DateHistogram.php

```diff
@@ -11,34 +9,6 @@
 class DateHistogram extends Histogram
 {
     /**
-     * Set pre-rounding based on interval.
-     *
-     * @deprecated Option "pre_zone" is deprecated as of ES 1.5 and will be removed in further Elastica releases. Use "time_zone" instead
-     *
-     * @param string $preZone
-     *
-     * @return $this
-     */
-    public function setPreZone($preZone)
-    {
-        throw new DeprecatedException('Option "pre_zone" is deprecated as of ES 1.5 and will be removed in further Elastica releases. Use "time_zone" instead.');
-    }
-
-    /**
-     * Set post-rounding based on interval.
-     *
-     * @deprecated Option "post_zone" is deprecated as of ES 1.5 and will be removed in further Elastica releases. Use "time_zone" instead.
```

Comment:
We can directly use these comments for the changelog as I think they are quire good. Just replacing `deprecated` by `removed`.